data/sitesEN.json

```diff
@@ -395,6 +395,17 @@
     "destination_platform": "mediawiki",
     "destination_icon": "tearsofthemis.png"
   },
+  {
+  "id": "en-touhou",
+  "origin": "Touhou Fandom Wiki",
+  "origin_base_url": "touhou.fandom.com",
+  "origin_content_path": "/wiki/",
+  "destination": "Touhou Wiki",
+  "destination_base_url": "en.touhouwiki.net/wiki/Touhou_Wiki",
```

Comment:
Just `en.touhouwiki.net`, no path please

Reply:
Oh, I did not notice the path AT ALL. Just modified!